piet-coregraphics/src/lib.rs

```diff
@@ -382,8 +382,7 @@ impl<'a> RenderContext for CoreGraphicsContext<'a> {
     ) {
         if let CoreGraphicsImage::NonEmpty(image) = image {
             if let Some(cropped) = image.cropped(to_cgrect(src_rect)) {
-                // TODO: apply interpolation mode
-                self.ctx.draw_image(to_cgrect(dst_rect), &cropped);
+                self.draw_image(&CoreGraphicsImage::NonEmpty(cropped), dst_rect, _interp);
```

Comment:
Sorry I missed this earlier, but please rename the `_interp` variable to `interp`. The underscore means that it is not used, but that is no longer true.

After that you can mark this PR as ready and I'll approve it. Thanks for your patience!

Reply:
Quite right, thank you. Fixed in 4f7dc7115e0ae71abf9312a9285ff880b76c1d57.

I had originally hoped to address linebender/druid#2245 with this branch as well but it seems that'll have to be done within druid so I'll mark this PR as ready now. Thanks for getting it merged!